.github/workflows/virtual_hardware.yml

```diff
@@ -71,7 +73,7 @@ jobs:
       uses: aws-actions/configure-aws-credentials@v1
       with:
         role-to-assume: ${{ secrets.AWS_ASSUME_ROLE }}
-        aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+        aws-region: ${{ env.AWS_DEFAULT_REGION }}
```

Comment:
Why is "env" used for AWS_DEFAULT_REGION and "secrets" for AWS_ASSUME_ROLE ?

Reply:
Hi Robert, there is no good reason to have `AWS_DEFAULT_REGION` as a secret. Better to use ENV directly.

Reply:
How about AWS_ASSUME_ROLE? "secret" or "env"?

Reply:
I thought about that, just for the sake to play safe, better to have the rest as secrets. 
But all the AWS info in the secrets is low risk IMO.